README.md

```diff
@@ -47,6 +47,15 @@ to be installed separately.
 - [PSI4](https://qiskit.org/ecosystem/nature/apidocs/qiskit_nature.second_q.drivers.psi4d.html), a chemistry program that exposes a Python interface allowing for accessing internal objects
 - [PySCF](https://qiskit.org/ecosystem/nature/apidocs/qiskit_nature.second_q.drivers.pyscfd.html), an open-source Python chemistry program
 
+The above codes can be used in a very limited fashion through Qiskit Nature.
+While this is useful for getting started and testing purposes, a better experience can be had in the reversed order of responsibility.
+That is, in a setup where the classical code runs the Qiskit Nature components.
+Such an integration currently exists for the following packages:
+
+- PySCF via [qiskit-nature-pyscf](https://github.com/qiskit-community/qiskit-nature-pyscf)
```

Comment:
In this case we could point at the docs ie https://qiskit-community.github.io/qiskit-nature-pyscf/ which itself has a link to the github (I know the latter has a link to the docs but maybe being taken there directly). In other places when referring to Qiskit Algorithms or whatever I have usually had it link to the docs.